create-keystone-app/starter/auth.ts

```diff
@@ -1,51 +1,63 @@
-/*
-Welcome to the auth file! Here we have put a config to do basic auth in Keystone.
-
-`createAuth` is an implementation for an email-password login out of the box.
-`statelessSessions` is a base implementation of session logic.
-
-For more on auth, check out: https://keystonejs.com/docs/apis/auth#authentication-api
-*/
+// Welcome to some authentication for Keystone
+//
+// This is using @keystone-6/auth to add the following
+// - A sign-in page for your Admin UI
+// - A cookie-based stateless session strategy
+//    - Using a User email as the identifier
+//    - 30 day cookie expiration
+//
+// This file does not configure what Users can do, and the default for this starter
+// project is to allow anyone - logged-in or not - to do anything.
+//
+// If you want to prevent random people on the internet from accessing your data,
+// you can find out how by reading https://keystonejs.com/docs/guides/auth-and-access-control
+//
+// If you want to learn more about how our out-of-the-box authentication works, please
+// read https://keystonejs.com/docs/apis/auth#authentication-api
 
+import { randomBytes } from 'crypto';
 import { createAuth } from '@keystone-6/auth';
 
-// See https://keystonejs.com/docs/apis/session#session-api for the session docs
+// see https://keystonejs.com/docs/apis/session for the session docs
 import { statelessSessions } from '@keystone-6/core/session';
 
+// for a stateless session, a SESSION_SECRET should always be provided
+//   especially in production (statelessSessions will throw if SESSION_SECRET is undefined)
 let sessionSecret = process.env.SESSION_SECRET;
-
-// Here is a best practice! It's fine to not have provided a session secret in dev,
-// however it should always be there in production.
-if (!sessionSecret) {
-  if (process.env.NODE_ENV === 'production') {
-    throw new Error(
-      'The SESSION_SECRET environment variable must be set in production'
-    );
-  } else {
-    sessionSecret = '-- DEV COOKIE SECRET; CHANGE ME --';
```

Comment:
We shouldn't do this, someone will be burnt :fire: by it, somwhere

Reply:
I guess the problem is this will force someone to login every time they update the schema